canonical-web-and-design/assets.ubuntu.com
# Set up sqlalchemy and alembic

## Done
- Set up project to use sqlalchemy as an ORM and alembic for data migrations.
- Add token views and CLI commands to manage tokens.

## QA

Follow both the simple SQLite instructions and the PostgreSQL instructions in the README:

https://github.com/canonical-web-and-design/assets.ubuntu.com/blob/8686b9b1181289910b11c2297d7ca98e785d2966/README.md#local-development

In both cases, try out the `flask token create`, `flask token list` and `flask token delete` commands.

Also, browse to http://127.0.0.1:8017/tokens?token=MYTOKEN, see your token listed.
